### DISCUSSION TITLE
generator-settings.pkl  file not found in %PATH%

### DISCUSSION THREAD

**Discussion by edson-dev** [apple/pkl#124] [Q&A] [answered]
I had the error `panic: exec: "pkl": executable file not found in %PATH%`, I have installed multiple times the pkl-go and pkl-gen-go , but it looks like it searches for a `generator-settings.pkl` file. I tried create the empty file, and searched a examples, and nothing work, i mean, looks like the run dont find the file in the current folder. Anyone know how to solve it?

**Comment by bioball** [comment, marked as answer]
The `generator-settings.pkl` file is optional. That error message is telling you that it can't find our `pkl` CLI installed on your machine.

See our [installation instructions](https://pkl-lang.org/main/current/pkl-cli/index.html) for how to install `pkl`.
